agent/src/main/java/com/cloud/agent/Agent.java

```diff
@@ -396,17 +397,16 @@ private synchronized void scheduleHostLBCheckerTask(final long checkInterval) {
             hostLBTimer.cancel();
         }
         if (checkInterval > 0L) {
-            logger.info("Scheduling preferred host timer task with host.lb.interval=" + checkInterval + "ms");
+            logger.info("Scheduling preferred host timer task with host.lb.interval={}ms", checkInterval);
             hostLBTimer = new Timer("Host LB Timer");
             hostLBTimer.scheduleAtFixedRate(new PreferredHostCheckerTask(), checkInterval, checkInterval);
         }
     }
 
     public void scheduleWatch(final Link link, final Request request, final long delay, final long period) {
         synchronized (_watchList) {
-            if (logger.isDebugEnabled()) {
-                logger.debug("Adding a watch list");
-            }
+            logger.debug("Adding a watch list");
```

Comment:
We could rewrite this log to add the task that is being added on the watchList.

Reply:
logger.debug("Adding task with request: {} to watch list", request.toString());
something like this?